variables.tf

```diff
@@ -18,6 +18,12 @@ variable "plan_validation" {
   default     = true
 }
 
+variable "existing_key_protect_instance_guid" {
```

Comment:
As per https://github.com/terraform-ibm-modules/terraform-ibm-cos/pull/265#issuecomment-1471869957 this variable name is being changed in COS. Its a breaking change for COS module, but since it doesn't exist in this module yet, we can call it what we want. Lets ensure we name it whatever we are going to name it for the COS module. Check with @daniel-butler-irl and @shemau

Reply:
In the context of using hyper protect rather than key protect in this module it would be better to use `kms_key_crn` albeit the breaking change. Also, this Postgres module is not consumed anywhere (except the ge dev account repo where we don't use this variable) so we might proceed with this change.

Reply:
Well the module is public, so it could be being consumed by anyone. However the breaking change will not destroy any resources so I am fine to go with it and we include the change in the release notes